src/Features/Core/Portable/MoveToNamespace/MoveToNamespaceResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.MoveToNamespace
+{
+    internal class MoveToNamespaceResult
+    {
+        public static MoveToNamespaceResult Failed = new MoveToNamespaceResult(succeeded: false);
```

Comment:
Oh, my comment on `MoveToNamespaceOptionsResult`, I meant something like this.

Reply:
Cancelled already exists, but I hid the constructor for it